bgpd/bgp_vty.c

```diff
@@ -11639,6 +11692,9 @@ DEFPY(show_ip_bgp_summary, show_ip_bgp_summary_cmd,
 			as = (as_t)atoi(argv[idx + 1]->arg);
 	}
 
+	if (!argv_find(argv, argc, "terse", &idx))
+		SET_FLAG(show_flags, BGP_SHOW_OPT_DETAIL);
```

Comment:
This breaks backward compatibility actually. BGP_SHOW_OPT_DETAIL is used to show only detailed JSON output. Or what's the reason for setting this flag?

Reply:
OK I have added BGP_SHOW_OPT_TERSE